### PR TITLE
feat(weed.move): Copy and move: add a speed limit parameter

### PR DESCRIPTION
**Pull request by qzhello**
# What problem are we solving?
When using the shell for move and copy, we want to limit the speed, not limited to the default parameter (compactionBytePerSecond) of the volume server.
We will evaluate the movement speed of different collections to ensure safe and fast range.

# How are we solving the problem?
Increase the rate limit parameter when requesting, if it is 0, use the default configuration of the volume server.
```
message VolumeCopyRequest {
    uint32 volume_id = 1;
    string collection = 2;
    string replication = 3;
    string ttl = 4;
    string source_data_node = 5;
    string disk_type = 6;
    // here
    int64 compaction_byte_per_second = 7;
}
```

# Checks
- [ ] I have added unit tests if possible.
- [ ] I will add related wiki document changes and link to this PR after merging.

### CODE REVIEW

File: weed/shell/command_volume_move.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ func (c *commandVolumeMove) Do(args []string, commandEnv *CommandEnv, writer io.
 	sourceNodeStr := volMoveCommand.String("source", "", "the source volume server <host>:<port>")
 	targetNodeStr := volMoveCommand.String("target", "", "the target volume server <host>:<port>")
 	diskTypeStr := volMoveCommand.String("disk", "", "[hdd|ssd|<tag>] hard drive or solid state drive or any tag")
+	compactionBytePerSecond := volMoveCommand.Int64("compactionBytePerSecond", 0, "limit the speed of copy")
```
Comment (chrislusf) [inline weed/shell/command_volume_move.go]: this is not compaction. how about `ioBytePerSecond`? (Same applies to the variable names.)

File: weed/server/volume_grpc_copy.go [hunk 2]
```diff
@@ -108,7 +108,14 @@ func (vs *VolumeServer) VolumeCopy(req *volume_server_pb.VolumeCopyRequest, stre
 		nextReportTarget := reportInterval
 		var modifiedTsNs int64
 		var sendErr error
-		if modifiedTsNs, err = vs.doCopyFile(client, false, req.Collection, req.VolumeId, volFileInfoResp.CompactionRevision, volFileInfoResp.DatFileSize, dataBaseFileName, ".dat", false, true, func(processed int64) bool {
+		var compactionBytePerSecond = func() int64 {
```
Comment (chrislusf) [inline weed/server/volume_grpc_copy.go]: why need the closure here?

File: weed/shell/command_volume_move.go (hunk 1 above)
Comment (qzhello) [reply weed/shell/command_volume_move.go]: I agree with you.

File: weed/server/volume_grpc_copy.go (hunk 2 above)
Comment (qzhello) [reply weed/server/volume_grpc_copy.go]: I'm sorry. What does that mean?

Comment (chrislusf) [reply weed/server/volume_grpc_copy.go]: the wrapping function is not necessary

**Pull request merged**